spec/unit/models/thread.spec.ts

```diff
@@ -510,6 +510,45 @@ describe("Thread", () => {
                 // And the first message has been edited
                 expect(secondLastEvent.getContent().body).toEqual("edit");
             });
+
+            it("Adds edits fetched on demand to the thread timeline and applies them", async () => {
```

Comment:
```suggestion
            it("applies edits fetched on demand", async () => {
```

I'd be more comfortable if this test's title were honest about what it's really testing at the moment ^^

Reply:
Please forgive me for not doing this.. I promise I'll merge https://github.com/matrix-org/matrix-js-sdk/pull/3398 very very soon.

It was going to topple my tower of PRs :-)